### PR TITLE
add images

### PR DESCRIPTION
**Pull request by Loganand-123**

### CODE REVIEW

File: src/Services.jsx [hunk 1]
```diff
@@ -15,31 +15,31 @@ export default function Services() {
     {
       className: "card1",
       id: 1,
-      img: "./assets/images/homevector.png",
+      img: "./assets/images/Web_Development.png", //public\assets\images\Web_Development.png
```
Comment (praveensumathi) [inline src/Services.jsx:18]: remove this comments

File: src/Technology.css [hunk 2]
```diff
@@ -1,8 +1,7 @@
-.container{
+/* .container{ */
```
Comment (praveensumathi) [inline src/Technology.css:1]: remove this

File: src/ChooseUs.jsx [hunk 3]
```diff
@@ -141,49 +140,53 @@ function ChooseUs() {
...
                 <Box sx={{ float: "right", display: "block" }}>
                   <Typography
                     mb={1}
                     className="heading"
-                    fontWeight={"500"}
-                    color="black"
-                    style={{ fontSize: matches ? "4vw" : "2.2vw" }}
-                    // variant="h6"
-                    // component="div"
+                    // fontWeight={"500"}
```
Comment (praveensumathi) [inline src/ChooseUs.jsx:170]: remove this

File: src/Services.jsx (hunk 1 above)
Comment (Loganand-123) [reply src/Services.jsx:18]: remove the comments lines

File: src/Technology.css (hunk 2 above)
Comment (Loganand-123) [reply src/Technology.css:1]: remove this lines

File: src/ChooseUs.jsx (hunk 3 above)
Comment (Loganand-123) [reply src/ChooseUs.jsx:170]: change this codes

**Pull request merged**